website/blog/2020-06-19-getting-started-and-aria.md

```diff
@@ -0,0 +1,213 @@
+
+---
+
+[PrestoDB](https://prestodb.io) recently released a set of experimental features under their Aria project in order to increase table scan performance of data stored in ORC files via the Hive Connector.  In this post, we'll check out these new features at a very basic level using a test environment of PrestoDB on Docker.  To find out more about the Aria features, you can check out the [Facebook Engineering](https://engineering.fb.com/data-infrastructure/aria-presto/) blog post which was published June 2019.
+
+Presto is a massively parallel processing (MPP) SQL execution engine.  The execution engine is decoupled from data storage, and the project contains numerous plugins, called _Connectors_, that provide the Presto engine with data for query execution.  Data is read from the data store, then handed to Presto where it takes over to perform the operations of the query, such as joining data and performing aggregations.  This decoupling of data storage and execution allows for a single Presto instance to query various data sources, providing a very powerful federated query layer.  There are many connectors available for Presto, and the community regularly provides additional connectors for data stores.
+
+The Hive Connector is often considered the standard connector for Presto.  This connector is configured to connect to a Hive Metastore, which exposes metadata about the tables defined in the Metastore.  Data is typically stored in HDFS or S3, and the Metastore provides information about where the files are stored and in what format; typically ORC but other supported formats like Avro and Parquet are supported.  The Hive connector allows the Presto engine to scan data from HDFS/S3 in parallel into the engine to execute your query.  [ORC (Optimized Row Columnar)](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+ORC) format is a very standard and common format for storing data, as it provides good compression and performance.
+
+Presto has two core services for executing queries. A _Coordinator_, which is responsible for query parsing and scheduling (among other things), and many _Workers_ which execute the queries in parallel.  The Coordinator can also act as a Worker, though it is not used for production environments.  Since we're playing with Presto here, we'll just use one node to act as both a Coordinator and Worker.  More detailed documentation, including installation details, can be found [here](https://prestodb.io/docs/current/).
+
+Let's take a look at getting a Docker image together for Presto (though they already exist on Dockerhub, e.g. [ahanaio/prestodb-sandbox](https://hub.docker.com/r/ahanaio/prestodb-sandbox)).  We can see below how relatively easy it is to get Presto up and running.  We download Presto, copy some configuration files in a local `etc` directory into the image, and specify an entry point to run the server.
```

Comment:
Starting from here I would suggest moving this into the docs under the deployment section

Reply:
FYI when I comment on a paragraph it puts the comment under the paragraph I am commenting on so I am including the paragraph above the comment.